backend/kotlin/baby-project-ktor/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = "babyProject"
```

Comment:
IMO that does not matter since the Directory is called `baby-project-ktor` and that basically just says what the artifact name should be, It can totally be `babyProjectKtor`

Reply:
In [backend/kotlin/baby-project-spring-boot/settings.gradle.kts](backend/kotlin/baby-project-spring-boot/settings.gradle.kts) `rootProject.name` is `"BabyProject"`, i.e. with PascalCase, but here camelCase is used. I don't know which one is correct, but I think it should be consistent. 🙂

Reply:
According to my IDE a package name should not be PascalCase.

And that name is generated when you greate a Ktor package

Reply:
But ether way, the name is kinda irrelevent because its under the path 'baby-project-ktor' i can change it to babyProjectKtor but it could just as well be fooBar